pandas/core/common.py

```diff
@@ -1857,23 +1857,84 @@ def _maybe_box_datetimelike(value):
 
 _values_from_object = lib.values_from_object
 
-
-def _possibly_convert_objects(values,
-                              datetime=True,
-                              numeric=True,
-                              timedelta=True,
-                              coerce=False,
+# TODO: Remove in 0.18 or 2017, which ever is sooner
+def _possibly_convert_objects(values, convert_dates=True,
```

Comment:
How about we put ``_possibly_convert_objects`` and ``_soft_convert_objects`` in ``pandas/tools/convert.py`` (new file)
``common.py`` is just so huge. We can eventually move more stuff their as well. It will have to import ``common`` for now.